sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -514,65 +587,84 @@ def _from_generated(cls, model):
             status=model.model_info.status,
             created_on=model.model_info.created_date_time,
             last_updated_on=model.model_info.last_updated_date_time,
-            extracted_fields=FormFields._from_generated(model.keys),
-            training_info=TrainingInfo._from_generated(model.train_result)
+            models=CustomFormSubModel._from_generated_unlabeled(model)
+            if model.keys else CustomFormSubModel._from_generated_labeled(model),
+            errors=FormRecognizerError._from_generated(model.train_result.errors) if model.train_result else [],
+            training_documents=TrainingDocumentInfo._from_generated(model.train_result)
+            if model.train_result else None
         )
 
 
-class TrainingInfo(object):
-    """Training information about the custom model.
+class CustomFormSubModel(object):
+    """Represents the submodel with model accuracy,
+    any errors while training the model, fields that were
```

Comment:
should we delete the errors section from this description?

Reply:
Ah yes, I haven't updated the docstrings after all the design changes. Thanks for point this out!

Reply:
updated